src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="page">
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
The class 'stopwatch--speed-up' is a modifier according to BEM, but there is no block or element that it modifies in your HTML. If it's not used in the CSS or JavaScript, consider removing it. Otherwise, ensure it modifies a relevant block or element.